### DISCUSSION TITLE
part-ii/2017-66/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#149] [Question comments]
Part II 2017 Further Complex Methods: Paper 1, Section II, E 

https://questions.tripos.org/part-ii/2017-66/

**Comment by acornnroca** [comment]
<details>
<summary>Solution</summary>

Take collapsed Hankel contour for Re(s) > 1, then extend to not-collapsed to show analytic continuation.

Find residues to calculate $$\zeta_R(-1) = -\frac{1}{12}$$


</details>
